Resources/config/value_resolver.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" ?>
+
+<container xmlns="http://symfony.com/schema/dic/services"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
+
+    <services>
+        <service id="doctrine_mongodb.odm.entity_value_resolver" class="Symfony\Bridge\Doctrine\ArgumentResolver\EntityValueResolver">
+            <argument type="service" id="doctrine_mongodb" />
+            <argument type="service" id="doctrine_mongodb.odm.entity_value_resolver.expression_language" on-invalid="ignore" />
+            <tag name="controller.argument_value_resolver" priority="110">
+                <attribute name="name">Symfony\Bridge\Doctrine\ArgumentResolver\EntityValueResolver</attribute>
```

Comment:
should be:
`<tag name="Symfony\Bridge\Doctrine\ArgumentResolver\EntityValueResolver" priority="110">controller.argument_value_resolver</tag>`

Reply:
Actually NO: this name would conflict with https://github.com/doctrine/DoctrineBundle/blob/2.9.x/Resources/config/orm.xml#L194 so we need to find another one

Reply:
On second though, this name might be the right one. It should allow `#[MapEntity]` to work correctly. I don't know how one would expect this to resolve the conflict when there is both ORM + MongoDB but I'd be tempted to say this should be handled inside `EntityValueResolver`.

Reply:
IIUC using Symfony 6.3 (with pinned resolvers), `#[MapEntity]` extending `ValueResolver` will set `EntityValueResolver::class` as resolver, but there won't be any resolver named `EntityValueResolver::class`, even if you are only using ORM, there will be `doctrine.orm.entity_value_resolver` but not `EntityValueResolver::class`, so using `#[MapEntity]` without specifying the resolver will probably fail?

Reply:
The resolver it registered under its FQCN, see name attribute on the tag in the XML I linked above.